### ISSUE TITLE
create an iOS version and make it available on test flight please

### ISSUE THREAD

**Issue by Dennisl123** [diamondStar35/top_speed#119]
Hi, can you create an iOS version and have it available on test flight?  It should be able to work with Voiceover running.  Thanks.

**Comment by diamondStar35** [comment]
@Dennisl123 
It is impossible for me to create an IOS version and make it available on the Test Flight. The only thing I can do is create an unsigned IOS version, and I can't even test it.
Please contact Apple support or go look at their stupid pricing model to learn more why such individual developers can't make their apps available publicly.
Also please do not use issues for this. It's better to open a discussion because this is not at all a bug.
